server/src/enum.ts

```diff
@@ -3,6 +3,28 @@ export enum AuthType {
   OAUTH = 'oauth',
 }
 
+export enum ImmichCookie {
```

Comment:
I was hoping js supports emotes in variable names but unfortunately that's not the case :( (`Immich🍪`)

Reply:
........